cdc_rsync/cdc_rsync_client.cc

```diff
@@ -260,6 +260,12 @@ absl::Status CdcRsyncClient::StartServer() {
     return SetTag(MakeStatus("Redeploy server"), Tag::kDeployServer);
   }
 
+  status = Socket::Initialize();
+  if (!status.ok()) {
```

Comment:
I assume you didn't use `RETURN_IF_ERROR` because the surrounding code doesn't use it. Have you considered changing that? It would save a lot of LoC and make the code more readable.

Reply:
Considered, yes! It hasn't quite crossed the threshold for me. I'm a bit afraid of causing some subtle bugs like missing a tag and such.